python/ray/experimental/actor_pool.py

```diff
@@ -0,0 +1,238 @@
+import ray
+import time
```

Comment:
```suggestion
```

Remove this line to avoid the following lint error:

```
./python/ray/experimental/actor_pool.py:2:1: F401 'time' imported but unused
```